setup/steps/install_milk_and_cacao.sh

```diff
@@ -72,5 +72,7 @@ else
   make_on_data_array "cacao-${MAGAOX_ROLE,,}" /opt/MagAOX
   sudo ln -sf "/opt/MagAOX/cacao-${MAGAOX_ROLE,,}" /opt/MagAOX/cacao
 fi
-log_info "Making /opt/MagAOX/cacao/ owned by xsup:magaox"
-sudo chown -R xsup:magaox /opt/MagAOX/cacao/
+if grep -q '^xsup:' /etc/passwd ; then
```

Comment:
These variables should be available since this script sources `_common.sh`: https://github.com/magao-x/MagAOX/blob/dev/setup/_common.sh#L5-L18 . It should be possible to just change `sudo chown -R xsup:magaox /opt/MagAOX/cacao/` to `sudo chown -R $instrument_user:$instrument_group /opt/MagAOX/cacao/`

If I'd used `$instrument_user` and `$instrument_group` in the first place there might not have been an issue, so that's my mistake.

Reply:
Are you sure this is independent of xsup?

Reply:
done